src_c/cdrom.c

```diff
@@ -88,7 +88,7 @@ static PyObject *
 get_count(PyObject *self)
 {
     CDROM_INIT_CHECK();
-    return PyInt_FromLong(SDL_CDNumDrives());
+    return PyLong_FromLong(SDL_CDNumDrives());
```

Comment:
cdrom and overlay should probably just be removed at this point.

Reply:
yeah, but that's probably for another PR hehe